### DISCUSSION TITLE
Zabbix template files

### DISCUSSION THREAD

**Discussion by mberthault** [selivan/https-ssl-cert-check-zabbix#23] [Ideas]
Hi,
I have created some template files for your script.
How can I send them to you for review and integration?

**Comment by selivan** [comment]
Hi, just use a Pull Request.

**Comment by iRomanyshyn** [comment]
Hi, @selivan! I made my own advanced template for Zabbix 5. Check it on my repo https://github.com/iRomanyshyn/Zabbix_SSL_TLS_check and I do allow you to merge it with this repo.
